src/buybutton.js

```diff
@@ -1,16 +1,18 @@
-import ShopifyBuy from 'shopify-buy/polyfilled';
+// import ShopifyBuy from 'shopify-buy/polyfilled';
+import ShopifyBuy, {Config} from 'shopify-buy';
 import UI from './ui';
 import productTemplates from './templates/product';
 
 window.ShopifyBuy = window.ShopifyBuy || ShopifyBuy;
+window.Config = window.Config || Config;
 
 ShopifyBuy.UI = window.ShopifyBuy.UI || {
   domains: {},
 
-  init(client, integrations = {}, styleOverrides) {
-    const domain = client.config.domain;
+  init(client, config, integrations = {}, styleOverrides) {
```

Comment:
The client has a subset of the previous config. This isn't a new concept, so you're totes right. keep the signature the same.